runtime/j9vm/valhallavmi.cpp

```diff
@@ -137,6 +152,16 @@ newArrayHelper(JNIEnv *env, jclass componentType, jint length, bool isNullRestri
 		goto done;
 	}
 
+	/* Set initial values. */
+	if (isNullRestricted) {
+		for (int index = 0; index < length; index++) {
+			/* It's not necessary to copy the inital value since a value class
```

Comment:
Typo "inital" --> "initial".

The code is copying the initial value, but the comment says "not necessary to copy the inital value", which is a bit confusing.